docs/glossary.md

```diff
@@ -101,9 +101,9 @@ The Claim Mapper component is a pluggable component that extracts Claims from JS
 
 _Tags: [term](/tags/term)_
 
-#### [Cluster configuration](/clusters#cluster-configuration)
+#### [Cluster](/clusters)
 
-Cluster Configuration is the setup and configuration details of your Temporal Cluster, defined using YAML.
+A Cluster is a Temporal Server paired with Persistence and Visibility stores.
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
```

Comment:
This needed to change from "Cluster configuration" to "Temporal Service configuration" and the link stays the same for now.